examples/kitchen-sink/client/main.js

```diff
@@ -4,11 +4,15 @@ import * as layout from '@financial-times/dotcom-ui-layout'
 import * as appContext from '@financial-times/dotcom-ui-app-context'
 import * as tracking from '@financial-times/n-tracking'
 import * as ads from '@financial-times/n-ads'
+import { loadClientEmbed } from '@financial-times/dotcom-ui-client-embed'
 
 readyState.domready.then(() => {
   const flagsClient = flags.init()
   const appContextClient = appContext.init()
 
+  const customContext = loadClientEmbed('custom-context')
```

Comment:
As you can see it's simple at the moment but does not follow the `init` convention elsewhere + the `getAll` pattern.

Should we require the user to pass their ID to `init` or automagically find clientEmbeds when `init()` is called and then allow them to use the client to specify an embed?

Reply:
Would be nice if we can keep it matching the pattern.

I think including the ID as an argument for `init` will make it clear which embed is being retrieved, but means it will require an individual DOM query for each embed being retrieved (although that is what the code is currently doing, and there is only one extra embed being added to one app at the moment, so this is minimal).

Reply:
One other alternative would be this sort of pattern:

```
@pixelandpage would appreciate your thoughts on what the interface could look like here as well if you have time :)